src/shared/components/addSample/addSample.styles.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+export const Button = styled.button`
+  width: auto;
+  font-size: 16px;
+  background-color: darkred;
```

Comment:
unikaj kolorów definiowanych po nazwie => hex lub rgb(a) lub hsl

Reply:
tak wiem, to tymczasowe przyciski